### PR TITLE
Support proposal timestamps

### PR DESCRIPTION
**Pull request by blue-note**
- Adds 'created' and 'last_updated' timestamps
- changes "vote" query for proposals to "get_vote"
- modifies color in proposal list if proposal was updated in the past day

Image 8-23-22 at 5 09 PM
Image 8-23-22 at 6 06 PM

### CODE REVIEW

**Comment by baoskee** [comment]
Wow is this your first UI PR?!? @blue-note

File: packages/i18n/locales/pl/translation.json [hunk 1]
```diff
@@ -362,6 +362,7 @@
     "contractVersion": "Wersja kontraktu",
     "createADAO": "Utwórz Organizację",
     "createAProposal": "Utwórz propozycję ustawy",
+    "created": "",
```
Comment (baoskee) [inline packages/i18n/locales/pl/translation.json:365]: Why are these strings empty?

File: packages/proposal-module-adapter/adapters/cw-proposal-single/components/ProposalLine/Desktop.tsx [hunk 2]
```diff
@@ -52,5 +60,28 @@ export const ProposalLineDesktop = ({ className }: BaseProposalLineProps) => {
       <p className="col-span-3 truncate link-text">{proposal.title}</p>
       <p className="text-right truncate body-text">{expirationString}</p>
     </div>
-  )
+    </Tooltip>) : (<div
+    className={clsx(
+      'grid grid-cols-6 items-center p-4 text-sm rounded-lg transition bg-primary hover:bg-secondary',
+      {
+        'bg-gray-500/30': msSinceUpdated < 24 * 60 * 60 * 1000,
```
Comment (baoskee) [inline packages/proposal-module-adapter/adapters/cw-proposal-single/components/ProposalLine/Desktop.tsx:67]: hey there's code duplication here, why don't we extract this so in the future changes can be made in one place.

**Review by baoskee — APPROVED** [review]

File: packages/i18n/locales/pl/translation.json (hunk 1 above)
Comment (blue-note) [reply packages/i18n/locales/pl/translation.json]: oops forgot to revert this

File: packages/state/clients/cw-proposal-single.ts [hunk 3]
```diff
@@ -422,7 +424,7 @@ export class CwProposalSingleQueryClient
     voter: string
   }): Promise<VoteResponse> => {
     return this.client.queryContractSmart(this.contractAddress, {
-      vote: {
+      get_vote: {
```
Comment (NoahSaso) [inline packages/state/clients/cw-proposal-single.ts:427]: Memorializing discord conversation to make this conditional based on contract version!

File: packages/proposal-module-adapter/adapters/cw-proposal-single/components/ProposalLine/Mobile.tsx [hunk 4]
```diff
@@ -28,12 +28,14 @@ export const ProposalLineMobile = ({ className }: BaseProposalLineProps) => {
   )
 
   const expirationString = useProposalExpirationString()
+  const msSinceUpdated = new Date().getTime() - (new Date(Number(proposal.last_updated) / 1000000).getTime());
```
Comment (NoahSaso) [inline packages/proposal-module-adapter/adapters/cw-proposal-single/components/ProposalLine/Mobile.tsx:31]: (linter will not be happy with semicolon, run `yarn format` to fix)

File: packages/proposal-module-adapter/adapters/cw-proposal-single/components/ProposalLine/Desktop.tsx (hunk 2 above)
Comment (NoahSaso) [reply packages/proposal-module-adapter/adapters/cw-proposal-single/components/ProposalLine/Desktop.tsx:67]: Yeah, can we just set the contents to a variable and then either return them wrapped with the Tooltip or not? Like `return proposalModuleVersion === ContractVersion.V0_2_0 ? <Tooltip label="...">{contents}</Tooltip> : contents`

**Review by NoahSaso — CHANGES_REQUESTED** [review]
So awesome! Thanks for doing this :))

File: packages/proposal-module-adapter/adapters/cw-proposal-single/components/ProposalLine/Desktop.tsx (hunk 2 above)
Comment (blue-note) [reply packages/proposal-module-adapter/adapters/cw-proposal-single/components/ProposalLine/Desktop.tsx:67]: ok!

File: packages/state/clients/cw-proposal-single.ts [hunk 5]
```diff
@@ -322,7 +324,7 @@ export interface CwProposalSingleReadOnlyInterface {
   }) => Promise<ReverseProposalsResponse>
   proposalCount: () => Promise<ProposalCountResponse>
   // Modified since `vote` is duplicate enum.
-  getVote: ({
+  getVoteV1: ({
```
Comment (NoahSaso) [inline packages/state/clients/cw-proposal-single.ts:327]: Can you also add `getVoteV2` to the interface?

**Review by NoahSaso — CHANGES_REQUESTED** [review]
SWEET, one last minor change and then we're MERGING 🔥

**Review by NoahSaso — APPROVED** [review]
wooo!!

**Pull request merged**